wiki/Difficulties/osu!/Easy/en.md

```diff
@@ -60,7 +63,7 @@ Avoid using sliders that only use 1/2 of a beat in the timeline. The rhythm thes
 Being visually appealing
 ------------------------
 
-Since the map is considered to be easy, it is recommended to make the map visually appealing to look at because patterns will definitely be noticed and appreciated on an Easy. This can be achieved through [smooth looking curves](Making_Good_Sliders) and strong structured patterns.
+Since the map is considered to be easy, it is recommended to make the map visually appealing to look at because patterns will definitely be noticed and appreciated on an Easy. This can be achieved through [smooth looking curves](/wiki/Mapping_Techniques/Making_Good_Sliders) and strong structured patterns.
 
 There are more advanced things you can do with an Easy, so experiment and figure out what works best for you.
 
```

Comment:
line 77: the streams link is being merged with Rhythm ... I'm not sure if it will be final, but it will be `[Streams](/wiki/Mapping_Techniques/Rhythm)`

----
If you think I should undo this change, let me know.